### PR TITLE
feat: 49 save session with cookies and pages authorization

### PR DESCRIPTION
**Pull request by fgomensoro**

### CODE REVIEW

File: src/hooks/UseIsLoggedIn.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import { getCookie } from '../hooks/useCookie';
+
+const UseIsLoggedIn = () => {
+    const navigate = useNavigate();
+
+    const loggedInUser = getCookie('authToken');
+
+    useEffect(() => {
+        if (loggedInUser) {
```
Comment (elios11) [inline src/hooks/UseIsLoggedIn.jsx]: Couldn't we reutilize this one for redirecting to home or login depending on the result of loggedInUser?

File: src/hooks/UseIsLoggedIn.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { Outlet, useNavigate } from 'react-router-dom';
+
+import { getCookie } from '../hooks/useCookie';
```
Comment (elios11) [inline src/hooks/UseIsLoggedIn.jsx]: Make use of hooks alias

**Review by elios11 — CHANGES_REQUESTED** [review]

File: src/hooks/UseIsLoggedIn.jsx (hunk 1 above)
Comment (fgomensoro) [reply src/hooks/UseIsLoggedIn.jsx]: done

File: src/hooks/UseIsLoggedIn.jsx (hunk 2 above)
Comment (fgomensoro) [reply src/hooks/UseIsLoggedIn.jsx]: done

**Review by elios11 — APPROVED** [review]
awesome

**Review by elios11 — APPROVED** [review]

**Review by amililime — APPROVED** [review]

**Pull request merged**